lib/src/chip/chip-row/chip-row.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, HostBinding, OnInit, Input } from '@angular/core';
+import { ChipComponent } from '../chip.component';
+import { FilterButtonComponent } from '../filter-button/filter-button.component';
+
+@Component({
+    selector: 'hc-chip-row',
+    templateUrl: './chip-row.component.html',
+    styleUrls: ['../chip.component.scss']
+})
+export class ChipRowComponent implements OnInit {
+    @HostBinding('class') hostClass = 'chip-row';
```

Comment:
I believe that binding to the class attribute directly will whip out any classes the user may have put on there. I would verify and if so change it to
`@HostBinding('class.chip-row') hostClass = true;`